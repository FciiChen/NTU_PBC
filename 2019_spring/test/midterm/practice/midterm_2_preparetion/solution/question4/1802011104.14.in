這放燈守衛的位置，是半山一個突出的斷崖，只容兩三人呆著，平常用藤籃吊下來，殺了人一時也不會發覺。

　　鳳知微繼續攀山梁而過，從強光交織之下一片淡霧朦朧之中，也如一縷淡霧一般青煙直上。

　　她快速的落在了碧照廳之上的客房所在的崖上，這裡守衛也很森嚴，可惜守的都是正門，背崖的那一面無法看守。

　　找到自己休息的那間房，侍女還在門外打盹，她做成的被窩卷還在，看起來像是一個人安靜的睡著，鳳知微快速過去，將床單撕成長條連接在一起，系在沉重的床腳下，試了試力度，覺得幸虧自己不胖，隨即將頭髮紮緊，將繩索系在右腳踝上，推開後窗，一個倒翻，仰身落下。

　　她落羽般的身形在半空翻開一個流逸的弧度，像深青的夜色裡悄然綻開一朵神秘的曇花，底下交織的燈光到了此處，氤氳出一片月白色的朦朧霧氣，那朵花便開在霧氣裡星光中，柔曼自在的舒展開來。

　　閉上眼睛，在心中想了想碧照廳的佈局，鳳知微腳尖一點，控制著布繩落下的長度，在某處停了下來。

　　她後背無聲的滑在崖壁上，濕滑的夜露浸濕衣服和頭髮。

　　這種倒仰的姿勢雖然很危險被動，但是比雙手雙腳都攀在岩上要好，最起碼除了一隻腳，其餘都是自由的。

　　從入口到此處，一路說起來簡單，可隨便哪裡出了一點岔子，她便死無葬身之地，直到此刻，她一直提緊的心，才微微放下一點來。


 
　　因為她聽見了寧弈的聲音。

　　「……殿下放心，」寧弈似乎在表白，「……這事天知地知你知我知，從此便爛在下官肚裡，便是我家夫人，夢話也別想聽得一句……」

　　鳳知微靜靜聽著，唇角綻出一抹笑意。

　　是不會聽見夢話，都不睡在一床嘛。

　　腦海中突然掠過溫泉東池的一幕，鳳知微臉上一紅，趕緊收斂心神，將微熱的臉頰，貼在冰冷的崖壁上。

　　真難為寧弈這傢伙，明明什麼都不清楚，居然還能和二皇子周旋到現在。

　　「多虧了你啊知秋。」二皇子似乎在嘆息，「……不過是長寧藩那邊的舊部一點小舉動，竟然被當地官府鬧成了綠林嘯聚謀逆案，多虧你幫忙改了證詞，又把幾個關鍵人物給解決了……大理寺那邊……不會有問題吧？」

　　「殿下放心。」寧弈似乎在拍胸脯，信誓旦旦語氣堅決。

　　二皇子似乎低低笑了聲。


 
　　「啪。」

　　一聲似有若無的低響。

　　「啊——」

　　寧弈一聲驚呼。

　　崖壁上鳳知微心中一跳。

　　「殿下……殿下……你——」寧弈的聲音有些模糊的傳來，似乎氣喘甚急。

　　鳳知微手指摳在崖壁裡，面無表情，卻將自己又往下降了降，已經靠近碧照廳這間密室的窗邊。

　　窗戶開著，透過窗櫺上方一點光線，可以看見室內映在牆上的人影。

　　隱約二皇子在獰笑，步步上前，而寧弈捂著胸口，步步後退。

　　「我？我什麼？」二皇子指指寧弈，冷笑道，「我還沒問你，你是誰的人？」

　　鳳知微眼神一閃。

　　「殿下……此話何意……」寧弈驚惶的聲音傳來。

　　「你今夜來，得了誰的指示？」二皇子上前一步，「你怎麼跑到西池去，又從西池跑到東池？你怎麼知道機關開啟辦法？你去那裡，要做什麼？」

　　果然……還是知道了。

　　鳳知微眯起眼睛，心想山莊應該還有別的消息渠道，二皇子這是在懷疑這黃大人不是他陣營，卻沒有懷疑這黃大人本人。

　　「下官……不明白……殿下的意思……」寧弈被二皇子逼著，退到窗邊。

　　「你不用明白了。」二皇子獰笑，「本王也不想明白你，反正有沒有這事，你都注定要死……」

　　「啊——」

　　一聲低低的驚呼，窗戶被砰然一撞，人影翻落。

　　==

　　人影翻落。

　　一個倒身落崖的姿勢。

　　卻有人風一般一蕩而下，閃電般的伸手一撈。

　　半空中兩隻手緊緊攥在一起。

　　腳尖還勾著窗邊的寧弈一抬頭，便看見上方懸崖倒掛而下的笑臉。

　　那張臉，在星月之光和無涯蒼穹背景裡俯衝而下，瞬間撞入他的眼簾，那一瞬間，漫天的星光和山間的霧氣，還有他的整個人，都似凝聚在那雙浮波浩淼的眸子裡，和天地剎那同存。

　　她在這裡等他。

　　他眼神瞬間亮起無限光華，有驚喜有擔憂有種種般般難以言說的複雜情緒，卻在迎面而上眼神交匯的那一剎，只供她讀懂。

　　隨即他也一笑，鬆開了勾住窗口的腳尖。

　　原本打算借老二下手借勢翻出窗口裝死，他的腳尖一直勾著窗邊，此刻他決然放開。

　　此刻他只有一隻手在她手中。

　　只要她放開手，他便無可挽回的落入萬丈深淵。

　　他終於在這一剎，將自己交給她。

　　腳尖鬆開，他身子一傾，鳳知微手中一重，絲繩往下墜了墜。

　　一瞬間她浮光浩渺的眼神裡，有什麼東西，飛快的一閃而過。
BREAK