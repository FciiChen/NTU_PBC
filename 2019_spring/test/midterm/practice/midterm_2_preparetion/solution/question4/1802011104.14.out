鳳_微 10
二_子 9
似_在 3
勾_窗 3
崖_上 3
弈_聲 3
的_音 3
知_知 3
碧_廳 3
一_倒 2
